src/components/List/List.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './List.module.scss';
+import ListItem from './ListItem';
+
+const List = ({ items }) => (
+	<ul className={styles.itemsWrapper}>
+		{items.map((item, index) => (
+			<ListItem {...item} key={index} />
+		))}
+	</ul>
+);
+
+List.propTypes = {
+	items: PropTypes.array,
```

Comment:
Don't use `array`, use instead `arrayOf`. Using `array` is equivalent to no validation at all.